source/j.ui/TTUiInfo.cpp

```diff
@@ -208,12 +208,21 @@ TTErr TTUiInfo::Panel()
 {
 	t_atom a;
     
-    if (mObject->patcher_panel) {
+    if (mObject->patcher_panel)
+    {
+        // output a bang to do some specific actions before to open
+        t_object    *box;
```

Comment:
This `box` pointer should be initialized to `NULL` in case the obex lookup below fails.